test/integration/ReplicationTest.cc

```diff
@@ -0,0 +1,120 @@
+#define ELPP_THREAD_SAFE
+
+#include "zkwrapper.h"
+#include "zk_nn_client.h"
+#include "zk_dn_client.h"
+#include "ClientNamenodeProtocolImpl.h"
+#include "data_transfer_server.h"
+#include <thread>
+#include <vector>
+#include <gtest/gtest.h>
+#include <asio.hpp>
+#include "native_filesystem.h"
+#include <cstring>
+
+
+#include <easylogging++.h>
+
+INITIALIZE_EASYLOGGINGPP
+
+using asio::ip::tcp;
+using client_namenode_translator::ClientNamenodeTranslator;
+int num_threads = 4;
+int max_xmits = 100000;
```

Comment:
@nichhk change this back to something reasonable. I think it needs to be 2 for the test to "work."